### PR TITLE
Remove alpine

### LINKED ISSUE #26: Only support the debian base image ?
The dependency management in Alpine with version pinning is hell: older package versions often disappears from the repository.
Maybe we should drop the alpine image, and focus only on the Debian one ? 
Image size are quite similar, so there is no big advantage for Alpine here.

### PR DESCRIPTION
**Pull request by pyaillet**
Closes #26

### CODE REVIEW

**Comment by bgauduch** [comment]
Some changes needs to be done on the Docker Hub, as modifications on this PR (Dockerfile name for instance) have impact on automated builds.

It should also be synchronized with the CI PR #27

**Comment by pyaillet** [comment]
Not sure it's worth rebuilding the old images.
Maybe deprecate them in the README ?
A new build with debian only should be useful beforehand though.

**Comment by bgauduch** [comment]
Yeah it's not worth it, we will only keep the old tags on the registry.
In the readme, we could just remove them from the 'supported tags' entries, as they wil clearly not be supported anymore.
Yes it should be good 👍

File: Dockerfile [hunk 1]
```diff
@@ -6,23 +6,26 @@ ARG TERRAFORM_VERSION=0.12.9
 FROM debian:stretch-20190506-slim as terraform
 ARG TERRAFORM_VERSION
 RUN apt-get update
+# hadolint ignore=DL3015
 RUN apt-get install -y curl=7.52.1-5+deb9u9
-RUN apt-get install -y unzip=6.0-21+deb9u2
-RUN apt-get install -y gnupg=2.1.18-8~deb9u4
+RUN apt-get install -y unzip=6.0-21+deb9u2 --no-install-recommends
+RUN apt-get install -y gnupg=2.1.18-8~deb9u4 --no-install-recommends
```
Comment (bgauduch) [inline Dockerfile]: Well done for the CI, although it's only in stage 👍

**Comment by bgauduch** [comment]
For the record, I have disabled old autobuilds and have added new ones on master / tags for debian only.

**Comment by bgauduch** [comment]
Need to rebase once #32 is merged 👍

**Pull request merged**